python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -781,6 +781,42 @@ def test_write_batch(library_factory):
         assert read_batch_result[sym].metadata == "great_metadata" + str(sym)
         assert_frame_equal(read_batch_result[sym].data, original_dataframe)
 
+    for sym in range(num_symbols):
+        data_key_version = lib._nvs.read_index("symbol_" + str(sym))["version_id"][0]
+        assert data_key_version == 0
```

Comment:
Shouldn't these 3 lines be in the test below? And we should check all the values in the `version_id` column, not just the first one.

Reply:
oh yeah. What a lapses.

changed it and also done the check for all the segments